aea/runtime.py

```diff
@@ -81,10 +83,22 @@ def __init__(
         self._multiplexer: AsyncMultiplexer = self._get_multiplexer_instance()
         self._task_manager = TaskManager()
         self._decision_maker: Optional[DecisionMaker] = None
+        self._storage: Optional[Storage] = self._get_storage(agent)
 
         self._loop_mode = loop_mode or self.DEFAULT_RUN_LOOP
         self.main_loop: BaseAgentLoop = self._get_main_loop_instance(self._loop_mode)
 
+    @staticmethod
+    def _get_storage(agent) -> Optional[Storage]:
+        if agent.storage_uri:
+            return Storage(agent.storage_uri, threaded=True)
```

Comment:
should the mode be configurable?

Reply:
no, only threaded. sync calls from skills do not allow to share event loop.

Reply:
please add a comment in code so the next developer knows why it must be so.

Reply:
done